content/docs/pipelines/tutorials/build-pipeline.md

```diff
@@ -89,15 +89,8 @@ Follow these steps to start a notebook:
     * When Kubeflow is running, access the Kubeflow UI at a URL of the form
       `https://<deployment-name>.endpoints.<project>.cloud.goog/`.
 
-    * Click **Pipeline Dashboard** to access the pipelines UI.
-
-1. Click **Notebooks** in the left-hend menu. If this is the first time you've 
-  visited JupyterHub, you need to sign in. You can use any username and you can
-  leave the password blank. 
-  
-1. Click the **Spawn** button to create a new instance. After a few 
-  minutes, the Jupyter UI opens. You can switch to the JupyterLab UI by changing 
-  `/tree` to `/lab` in the URL.
+1. Follow the [Kubeflow notebooks setup guide](/docs/notebooks/setup/) to
+  create a Jupyter notebook server and open the Jupyter UI.
```

Comment:
Same comment regarding Jupyter Notebook as an application. Consider changing:

Jupyter notebook server

 to:

Jupyter Notebook server

Reply:
I'm going to stick with "Jupyter notebooks" for simplicity across the docs, as I think that will be clearer to our readers. It may come across as a little confusing to sometimes use "Notebook" and other times "notebook", though I do take your point about correctness here.